cg/cli/upload/vogue.py

```diff
@@ -182,43 +163,44 @@ def bioinfo(context, case_name, cleanup, target_load, dry):
     if target_load in ("raw", "all"):
         click.echo(click.style("----------------- UPLOAD UNPROCESSED -----------------------"))
         if not dry:
-            analysis_api.upload_vogue_api.load_bioinfo_raw(load_bioinfo_raw_inputs)
+            upload_vogue_api.load_bioinfo_raw(load_bioinfo_raw_inputs)
 
     if target_load in ("process", "all"):
         click.echo(click.style("----------------- PROCESS CASE -----------------------"))
         if not dry:
-            analysis_api.upload_vogue_api.load_bioinfo_process(load_bioinfo_raw_inputs, cleanup)
+            upload_vogue_api.load_bioinfo_process(load_bioinfo_raw_inputs, cleanup)
         click.echo(click.style("----------------- PROCESS SAMPLE -----------------------"))
         if not dry:
-            analysis_api.upload_vogue_api.load_bioinfo_sample(load_bioinfo_raw_inputs)
+            upload_vogue_api.load_bioinfo_sample(load_bioinfo_raw_inputs)
 
 
 @vogue.command("bioinfo-all", short_help="Load all bioinfo results into vogue")
 @click.option("--dry/--no-dry", is_flag=True, help="Dry run...")
 @click.pass_context
-def bioinfo_all(context, dry):
+def bioinfo_all(context: click.Context, dry: bool):
     """Load all cases with recent analysis and a multiqc-json to the trending database."""
 
-    if not context.obj.get("analysis_api"):
-        context.obj["analysis_api"] = MipDNAAnalysisAPI(context.obj)
-    analysis_api = context.obj["analysis_api"]
+    status_db: Store = context.obj.status_db
+    housekeeper_api: HousekeeperAPI = context.obj.housekeeper_api
 
-    cases = analysis_api.status_db.families()
+    cases: List[models.Family] = status_db.families().all()
     for case in cases:
-        case_name = case.internal_id
-        version_obj = analysis_api.housekeeper_api.last_version(case_name)
+        case_name: str = case.internal_id
+        version_obj: hk_models.Version = housekeeper_api.last_version(case_name)
         if not version_obj:
             continue
 
         # confirm multiqc.json exists
-        multiqc_file_obj = analysis_api.housekeeper_api.get_files(
-            bundle=case_name, tags=["multiqc-json"], version=version_obj.id
+        multiqc_file_obj: List[hk_models.File] = list(
+            housekeeper_api.get_files(
+                bundle=case_name, tags=["multiqc-json"], version=version_obj.id
+            )
```

Comment:
```suggestion
        multiqc_file_obj: List[hk_models.File] = (
            housekeeper_api.get_files(
                bundle=case_name, tags=["multiqc-json"], version=version_obj.id
            ).first())
```

Reply:
I really don't want to do like this. This will assume that something that you can call `.first()` on is returned. So if we change the API all of these stuff will break.